docs/extensions/api.txt

```diff
@@ -68,6 +68,12 @@ must have the following methods:
     Accepts a match object and returns an ElementTree element of a plain 
     Unicode string.
 
+* **`getExcludes()`**: 
+
+    Returns an array of tag names that are undesirable ancestors. The pattern 
+    should not match if it would cause the content to be a descendant of one 
+    of the tag names in the list.
```

Comment:
"Returns an ***array***... tag names in the ***list***." Let's be consistent here. And in Python we call them lists, not arrays.

Reply:
I work in many languages, so things blur together at times and I make mistakes like this.